src/commands/l-x.ts

```diff
@@ -90,7 +92,7 @@ Fetching L-4 versions woocommerce!
 
         for (const version of versions) {
           const releasesAdded = output.filter(
-            release => !isRC(release),
+            release => (includeRC || !isRC(release)),
```

Comment:
Curious why the applied fix is not the same as it is in https://github.com/woocommerce/grow/pull/66?